### ISSUE TITLE
The bug genie 4.x mautic integration

### ISSUE THREAD

**Issue by 4evermaat** [mautic/mautic#1468]
It would be great if mautic could import a lead if they communicate with the bug tracker and they are not already existing in mautic.
http://thebuggenie.org/
http://issues.thebuggenie.com/wiki/TheBugGenie%3ADevelopment%3AJsonAPI
http://issues.thebuggenie.com/wiki/TheBugGenie%3ADevelopment%3AModules
http://issues.thebuggenie.com/wiki/TheBugGenie%3AAddons 
One of the easier project management (bug trackers) out there that I've seen.

Mainly I was interested in allowing someone who submits an issue to be tracked as a lead and tagged a certain way. Also, you cam move that lead through the mautic campaign based on a number of factors, such as how many tickets he/she issues, the type of issue that was made (bug report, feature request, etc) which shows their modification, etc.

So basically be able to track the activities of what the person does when they (their email address/cookie/etc) are doing in bug genie and modify their milestone in a mautic campaign.
